web/components/answers/answers-panel.tsx

```diff
@@ -556,3 +604,49 @@ function Answer(props: {
     </Col>
   )
 }
+
+function LimitOrderBarChart({
+  limitOrders,
+  prob,
+  activeColor,
+}: {
+  limitOrders?: Array<LimitBet>
+  prob: number
+  activeColor: string
```

Comment:
I realize this is a more concise way of declaring and unpacking props. Maybe we should do it this way everywhere